js/repl/Repl.js

```diff
@@ -612,6 +613,12 @@ class Repl extends React.Component<Props, State> {
     UriUtils.updateQuery(payload);
   };
 
+  _onVersionChange = (e: Event) => {
+    this.state.babel.version = e.target.value;
```

Comment:
Any reason we're mutating state directly here instead of `this.setState`?

Reply:
No, I used to mutate state when I learned React since most of time they works. Bad habits.